src/webapp/js/components/modules-manage.js

```diff
@@ -22,6 +64,73 @@ export default class ModulesManage extends Component {
     }
   }
 
+  getNodeClassName(node) {
+    const classNames = [];
+
+    if (node.type !== 'MODULE') {
```

Comment:
Would not be more convinient here to have dict with key - node.type and value className?
eg.:
const nodeTypes = {
 nodeType1: 'class-name-1',
 nodeType2: 'class-name-2',
};
classNames.push(nodeTypes[node.type]);

Reply:
it's just one if, rest of the node type is not checked, they all have the same class

Reply:
ok